assets/javascripts/discourse/components/chat-selection-manager.js

```diff
@@ -11,17 +13,27 @@ import getURL from "discourse-common/lib/get-url";
 
 export default class AdminCustomizeColorsShowController extends Component {
   tagName = "";
-  @service router;
   chatChannel = null;
   selectedMessageIds = null;
   showChatQuoteSuccess = false;
   cancelSelecting = null;
 
+  @service router;
+  @alias("currentUser.staff") showMoveMessages;
+
   @computed("selectedMessageIds.length")
   get anyMessagesSelected() {
     return this.selectedMessageIds.length > 0;
   }
 
+  @action
+  moveMessagesToChannel() {
```

Comment:
Not sure about the name here, sounds like we are actually moving stuff, when we are just showing Ui at that point

Reply:
Yes good point, will change to `openMoveMessageModal` :+1: